client/Windows/wf_event.c

```diff
@@ -656,13 +696,18 @@ BOOL wf_scale_blt(wfContext* wfc, HDC hdc, int x, int y, int w, int h,
 	return TRUE;
 }
 
-static BOOL wf_scale_mouse_event(wfContext* wfc, rdpInput* input, UINT16 flags,
-                                 UINT16 x, UINT16 y)
+static BOOL wf_scale_mouse_pos(wfContext* wfc, UINT16* x, UINT16* y)
 {
 	int ww, wh, dw, dh;
 	rdpContext* context;
-	rdpSettings* settings = wfc->context.settings;
-	MouseEventEventArgs eventArgs;
+	rdpSettings* settings;
+       
+	if (!wfc || !x || !y)
+		return FALSE;
```

Comment:
unneeded checks these should be asserts

Reply:
@hardening API checks, this is not internal (client != lib)

Reply:
This is a local functions, just give me the case where one of these pointers could be NULL and you'll see that it implies that a caller didn't do the check job.